Libplanet/Net/NetMQTransport.cs

```diff
@@ -992,6 +983,32 @@ private void ValidateSender(Peer peer)
             }
         }
 
+        private Task RunPoller(NetMQPoller poller) =>
+            Task.Factory.StartNew(
+                () =>
+                {
+                    // Ignore NetMQ related exceptions during NetMQPoller.Run() to stabilize
+                    // tests.
+                    try
+                    {
+                        poller.Run();
+                    }
+                    catch (TerminatingException)
+                    {
+                        _logger.Error($"TerminatingException occurred in {nameof(poller)}");
```

Comment:
Why `"poller"` should be written as `nameof(poller)` instead?

Reply:
I guess there is no intention and it seems moved automatically. I'll fix.